src/components/CustomerAppeal/ReasonSelection/ReasonSelectionCard.tsx

```diff
@@ -66,16 +66,7 @@ export const ReasonSelectionCard: FunctionComponent<ReasonSelectionCard> = ({
         <SecondaryButton
           text="Cancel"
           onPress={() => {
-            showAlert({
-              ...defaultWarningProps,
-              title: "Cancel entry and scan another ID number?",
-              buttonTexts: {
-                primaryActionText: "Cancel entry",
-                secondaryActionText: "Keep"
-              },
-              visible: true,
-              onOk: onCancel
-            });
+            showWarnAlert(new Error("cancelEntry"), onCancel);
```

Comment:
Hmm rather than this, why don't we just rename this `showCancelEntryWarnAlert` like:

```
showCancelEntryWarnAlert(onCancel);
```

Since it's the only use of this show warn alert anyway?

Reply:
What if there are more warn alerts in the future? I feel that it is best to leave it as is, what do you think?

Reply:
I have refactored `showWarnAlert` to take in an `enum`, like so:
```suggestion
            showWarnAlert(WARNING_MESSAGE.CANCEL_ENTRY, onCancel);
```
Let me know if this is acceptable!